dsp/window/window_test.go

```diff
@@ -230,29 +258,31 @@ func TestGausWindowComplex(t *testing.T) {
 	t.Parallel()
 	const tol = 1e-6
 
-	for _, test := range gausWindowTests {
-		t.Run(fmt.Sprintf("%sComplex (sigma=%.1f)", test.name, test.sigma), func(t *testing.T) {
-			src := make([]complex128, 20)
-			for i := range src {
-				src[i] = complex(1, 1)
-			}
+	for _, test := range monoParamWindowTests {
+		if test.winType == gaussWin {
+			t.Run(fmt.Sprintf("%sComplex (sigma=%.1f)", test.name, test.param), func(t *testing.T) {
+				src := make([]complex128, 20)
+				for i := range src {
+					src[i] = complex(1, 1)
+				}
 
-			gaussian := GaussianComplex{test.sigma}
+				gaussian := GaussianComplex{test.param}
 
-			dst := gaussian.Transform(src)
-			if !equalApprox(dst, test.want, tol) {
-				t.Errorf("unexpected result for window function %q:\ngot:%#.6v\nwant:%#.6v", test.name, dst, test.want)
-			}
+				dst := gaussian.Transform(src)
+				if !equalApprox(dst, test.want, tol) {
```

Comment:
`equalApprox` is no longer needed to defined locally since we have `cmplxs.EqualApprox` now.